roles/ceph-rgw/tasks/docker/pre_requisite.yml

```diff
@@ -29,6 +29,13 @@
   tags:
     with_pkg
 
+- name: install six
```

Comment:
can you use a `with_items` loop to install pip's packages? so you can incorporate that with the docker-py installation